crates/fluvio-spu-schema/src/server/api.rs

```diff
@@ -42,6 +43,12 @@ pub enum SpuServerRequest {
     FileStreamFetchRequest(RequestMessage<FileStreamFetchRequest>),
     #[fluvio(tag = 5)]
     UpdateOffsetsRequest(RequestMessage<UpdateOffsetsRequest>),
```

Comment:
Big question is this intended between client and SPU or just between SPU to SPU.   If it is only for SPU to SPU then this should be moved to SPU private protocol

Reply:
between client and SPU

Reply:
Go it